source/extensions/transport_sockets/tls/ssl_socket.cc

```diff
@@ -124,6 +148,37 @@ Network::IoResult SslSocket::doRead(Buffer::Instance& read_buffer) {
   return {action, bytes_read, end_stream};
 }
 
+void SslSocket::complete() {
```

Comment:
Nit: It would be more readable if this method would be called `onPrivateMethodComplete` or something similar.

Reply:
That makes sense. Will fix.